source/slang/slang-check.cpp

```diff
@@ -161,12 +161,13 @@ namespace Slang
     }
 
     void checkTranslationUnit(
-        TranslationUnitRequest* translationUnit)
+        TranslationUnitRequest* translationUnit,
+        LoadedModuleDictionary& loadedModules)
     {
-        SharedSemanticsContext sharedSemanticsContext(
-            translationUnit->compileRequest->getLinkage(),
-            translationUnit->getModule(),
-            translationUnit->compileRequest->getSink());
+        SharedSemanticsContext sharedSemanticsContext(translationUnit->compileRequest->getLinkage(),
+                                                      translationUnit->getModule(),
+                                                      translationUnit->compileRequest->getSink(),
+                                                      &loadedModules);
```

Comment:
We don't have a lot of enforced coding-convention stuff, but this is one of the few cases where I'm inclined to get militant. The "indent all subsequent arguments to match the indent of the first" style is not robust against find-replace changes of type/function names, while the "indent all arguments by one level" style is.

I assume this was introduced by your editor, and that we should put more settings into the `.editorconfig` or whatever files we can to get better matching. I think the latest Visual Studio supports putting more of the code formatting settings in `.editorconfig` so we should try to leverage that.